### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-263.1][WIP] The main region should be marked for all the screens and content from where the main region starts should be conveyed to screen readers.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11244]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
The main region should be marked for all the screens and content from where the main region starts should be conveyed to screen readers.
When we develop UI using NGX component. In VPC mode, when navigated in R key, No main region along with its contents are read out. Even though we have defined page details in top level of every page

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Accessibility expert can help

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide more information?

**Comment by JyotiKumari1507** [comment]
@droshev  Lets discuss this in our next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by JyotiKumari1507** [comment]
@droshev  As per accessibility team, The main region should be marked for all the screens and content from where the main region starts should be conveyed to screen readers. In VPC mode, when navigated in R key, No main region along with its contents are read out.
Attaching screen reader screenshot for more details:
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/2830ed25-6603-4ca6-9327-bf5b466d3327">

**Comment by droshev** [comment]
most of the fields are not answered

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?

**Comment by JyotiKumari1507** [comment]
@divyamalhotra488 please share stackblitz example

**Comment by droshev** [comment]
@JyotiKumari1507 any updates?

**Comment by divyamalhotra488** [comment]
Hi @droshev, please find the stackblitz example here: https://stackblitz.com/edit/qnytqa?file=src%2Fapp%2Fdynamic-page-tabs-example.component.html

**Comment by divyamalhotra488** [comment]
Hi @droshev, please find the stackblitz example here: https://stackblitz.com/edit/qnytqa?file=src%2Fapp%2Fdynamic-page-tabs-example.component.html

**Comment by mikerodonnell89** [comment]
@JyotiKumari1507 @divyamalhotra488 Could you give some specifics about what exactly you expect to be read, and when, as the user navigates around the page? These core layout components should all be flexible enough that you can set aria attributes to get the screenreader to do whatever you'd like. But if we need to update this in our documentation, we will need more specifics

**Comment by droshev** [comment]
closing as there was no activity
